src/logic/otp2/controller.ts

```diff
@@ -57,6 +57,9 @@ interface Otp2TripPattern extends TripPattern {
 }
 
 interface AdditionalOtp2TripPatternParams {
+    // This is found on TripPatternParams too, but in our case it
```

Comment:
Kva er "This"? Antar det er berre `searchDate` du meiner og ikkje alle tre, men fint å spesifisere i tilfelle det blir omrokkering på linjer, osv.

Reply:
Bra innspill, endret.